app/models/like.rb

```diff
@@ -1,4 +1,12 @@
 class Like < ApplicationRecord
   belongs_to :user
```

Comment:
[REQUIRED] As explained in the comment model. Kindly stick to the use of `user` or `author` through the use of class_name. You are using user here but have a `foreign_key` set to `author_id`